yarn-project/circuit-types/src/interfaces/pxe.ts

```diff
@@ -107,6 +105,15 @@ export interface PXE {
    */
   getRecipient(address: AztecAddress): Promise<CompleteAddress | undefined>;
 
+  /**
+   * Rotates our master nullifier secret key by adding it to our keystore
```

Comment:
```suggestion
   * Rotates master nullifier keys.
```
Adding it to key store is an implementation detail irrelevant for the consumers of the interface.